app/views/application/mailer/_checklist.html.haml

```diff
@@ -0,0 +1,29 @@
+%table.email-w-full.email-checklist-wrapper-table{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+  %tr
+    %td.email-checklist-wrapper-td
+      %table.email-w-full.email-checklist-table{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+        %tr
+          %td.email-checklist-td
+            %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+              %tr
+                %td.email-checklist-icons-td
+                  %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+                    %tr
+                      %td.email-checklist-icons-checkbox-td
+                        - if defined?(checked) and checked
+                          = image_tag full_pack_url('media/images/mailer-new/welcome/checkbox-on.png'), alt: '', width: 20, height: 20
+                        - else
+                          = image_tag full_pack_url('media/images/mailer-new/welcome/checkbox-off.png'), alt: '', width: 20, height: 20
+                      %td.email-checklist-icons-step-td
+                        - if defined?(step_image_url)
+                          = image_tag step_image_url, alt: '', width: 40, height: 40
+                %td.email-checklist-text-td
+                  .email-desktop-flex
+                    %div
+                      - if defined?(title)
```

Comment:
This approach is fine -- but if we are very confident that the only places we ever use the partial do indeed pass in these local vars, we dont necessarily need the `defined?` checks.